packages/docusaurus-theme-classic/src/theme/CodeBlock/styles.module.css

```diff
@@ -12,6 +12,12 @@
   margin: 0;
 }
 
+.fileName {
+  padding: 4px 12px;
+  font-weight: bold;
+  text-decoration: underline;
```

Comment:
I think it would be better to separate these two blocks via the bottom border.

> code title
> -----------
> code

Reply:
This is a `boderBottom: '1px solid';` one with
```jsx
<pre className={classnames(className, styles.codeBlock)} style={style}>
```
![ss-for-docusaurus-scroll-codeblock](https://user-images.githubusercontent.com/29557494/75625760-bfb96400-5bf3-11ea-9fe7-92bbc416e6c1.gif)

The problem is 
1. I couldn't find easiest way to `filename div` width 100%.
2. That's why boder-bottom is lacking.
3. I found easy solution to set `text-decoration: underline;` and looks ok.